data/org.freedesktop.impl.portal.GlobalShortcuts.xml

```diff
@@ -0,0 +1,188 @@
+      <arg type="o" name="session_handle" direction="in"/>
+      <arg type="s" name="app_id" direction="in"/>
+      <arg type="a{sv}" name="options" direction="in"/>
+      <arg type="a{sv}" name="results" direction="out"/>
+      <annotation name="org.qtproject.QtDBus.QtTypeName.In3" value="QVariantMap"/>
+      <annotation name="org.qtproject.QtDBus.QtTypeName.Out0" value="QVariantMap"/>
+    </method>
+
+    <!--
+        ListShortcuts:
+        @handle: Object path for the #org.freedesktop.impl.portal.Request object representing this call
+        @session_handle: Object path for the #org.freedesktop.impl.portal.Session object representing the session
+        @results: Vardict with the results of the call
+
+        List the shortcuts registered in the @p handle session.
+
+        The following results get returned via the #org.freedesktop.portal.Request::Response signal:
+        <variablelist>
+          <varlistentry>
+            <term>shortcuts a{sv}</term>
+            <listitem><para>
+              The different shortcut ids that have been registered mapped to user-readable descriptions for the app to present that describes the shortcut.
+            </para></listitem>
+          </varlistentry>
+      </variablelist>
+    -->
+    <method name="ListShortcuts">
+      <arg type="o" name="handle" direction="in"/>
+      <arg type="o" name="session_handle" direction="in"/>
+      <arg type="a{sv}" name="results" direction="out"/>
+      <annotation name="org.qtproject.QtDBus.QtTypeName.Out0" value="QVariantMap"/>
+    </method>
+
+    <!--
+        Bind:
+        @handle: Object path for the #org.freedesktop.impl.portal.Request object representing this call
+        @session_handle: Object path for the #org.freedesktop.impl.portal.Session object representing the session
+        @shortcut_id: The identifier of the shortcut we intend to register
+        @results: Vardict with the results of the call
+
+        Allows to configure the binding the global shortcut named @p shortcut_id within the @session_handle session to a new shortcut.
+
+        The following results get returned via the #org.freedesktop.portal.Request::Response signal:
+        <variablelist>
+          <varlistentry>
+            <term>description</term>
+            <listitem><para>
+              The user-readable description of the trigger that the portal selected. This is intended for the app to be able to render the shortcut into the UI, allowing the user to learn the shortcut
+            </para></listitem>
+          </varlistentry>
+      </variablelist>
+    -->
+    <method name="Bind">
+      <arg type="o" name="handle" direction="in"/>
+      <arg type="o" name="session_handle" direction="in"/>
+      <arg type="s" name="shortcut_id" direction="in"/>
+      <arg type="a{sv}" name="results" direction="out"/>
+      <annotation name="org.qtproject.QtDBus.QtTypeName.Out0" value="QVariantMap"/>
+    </method>
```

Comment:
With the introduction of `BindShortcuts`, I think this method is unecessary now. People can use `BindShortcuts` for all cases, including individual shortcuts.

Reply:
Yes, I'll have to agree. In my PoC this proved to be harder to implement than I had hoped.